### ISSUE TITLE
Dataset Issues

### ISSUE THREAD

**Issue by Namonamos** [robingg1/PoseTraj#2]
I'd like to reproduce your work. Your training script includes `video_folder` and `validation_image_folder`:
 --video_folder="../render_objaverse_10k" \
 --validation_image_folder="../dataset/objaverse_val" \

<img width="624" height="132" alt="Image" src="https://github.com/user-attachments/assets/1a17e854-5c09-44d2-8941-788b94b9d9b4" />

Is the pre-generated dataset you provided the complete render_objaverse_10k dataset, or do I need to render PoseTraj-10k myself? 
Also, how can I obtain the objaverse_val dataset? Thank you for your response.

**Comment by robingg1** [comment]
Yes, the pre-built dataset is the complete 10k dataset, each split contains 5k videos. During my own training, the validation data is sampled from training set only to visualize the training performance along steps. If you want to construct a separate test set for metric computation on synthetic data, you can follow the rendering instructions to produce more data. The reported metric in the paper of synthetic dataset does the same.

**Comment by Namonamos** [comment]
OK, thank you for your reply.

**Comment by Namonamos** [comment]
Is the model_out_vipseg_ft/checkpoint-100000 on Hugging Face still not uploaded?

**Comment by robingg1** [comment]
Yes, I will upload one within this week.

**Comment by Namonamos** [comment]
Thank you.
